src/render/PanelHeaderProvider.js

```diff
@@ -134,4 +135,14 @@ function isConditionalFlow(element) {
   }
 
   return businessObject.conditionExpression && is(sourceBusinessObject, 'bpmn:Activity');
+}
+
+
+// helpers //////////
+function isPlane(element) {
+
+  // Backwards compatibility for bpmn-js<8
+  var di = element && (element.di || getBusinessObject(element).di);
```

Comment:
```suggestion
  const di = element && (element.di || getBusinessObject(element).di);
```

Reply:
:facepalm: To much time spent in `bpmn-js` ES5 land

Reply:
Fixed it with [ecd47d6](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/567/commits/ecd47d661d8d365aec19eca82585da9810a5f3c4)